cli/src/regtest.rs

```diff
@@ -1,9 +1,9 @@
-use std::process::Command;
```

Comment:
Why remove this?

Reply:
Because there's a another `Command` `tokio::process::Command` and I want to preempt confusion.